specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -1663,6 +1669,248 @@
+          "200": {
+            "description": "OK -- Put local user successfully.",
+            "schema": {
+              "$ref": "#/definitions/LocalUser"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "LocalUsers"
+        ],
+        "operationId": "LocalUsers_Delete",
+        "description": "Deletes the local user associated with the specified storage account.",
+        "x-ms-examples": {
+          "DeleteLocalUser": {
+            "$ref": "./examples/LocalUserDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/Username"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Delete the local user successfully."
+          },
+          "204": {
+            "description": "No Content -- The local user does not exist."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/localUsers/{username}/listKeys": {
+      "post": {
+        "tags": [
+          "LocalUsers"
+        ],
+        "operationId": "LocalUsers_ListKeys",
+        "description": "List the local user keys.",
+        "x-ms-examples": {
+          "ListLocalUserKeys": {
+            "$ref": "./examples/LocalUserListKeys.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/Username"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Get the local user keys successfully.",
+            "schema": {
+              "$ref": "#/definitions/LocalUserKeys"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/localUsers/{username}/regeneratePassword": {
+      "post": {
+        "tags": [
+          "LocalUsers"
+        ],
+        "operationId": "LocalUsers_RegeneratePassword",
```

Comment:
Please indicate which password will be regenerated in the API description?
Will only SshPassword be regenerated? Or SharedKey also be regenerated? (SshAuthorizedKeys is set by user, per my understanding it should not be regenearte. Please confirm)

Reply:
It will only regenerate sshPassword. To regenerate sharedKey, user will need to set hasSharedKey to false and true.